src/peft/tuners/lora/model.py

```diff
@@ -258,6 +258,10 @@ def _create_new_module(lora_config, adapter_name, target, **kwargs):
         else:
             target_base_layer = target
 
+        megatron_core = None
+        if lora_config.megatron_config:
+            megatron_core = __import__(lora_config.megatron_core, fromlist=["tensor_parallel", "transformer"])
```

Comment:
So I checked the [Python docs](https://docs.python.org/3/library/functions.html#import__) and they mention that:

> This is an advanced function that is not needed in everyday Python programming, unlike [importlib.import_module()](https://docs.python.org/3/library/importlib.html#importlib.import_module).

It's not quite clear to me what makes this "advanced", but it would probably be better to switch to `importlib.import_module`.

Reply:
> So I checked the [Python docs](https://docs.python.org/3/library/functions.html#import__) and they mention that:
> 
> > This is an advanced function that is not needed in everyday Python programming, unlike [importlib.import_module()](https://docs.python.org/3/library/importlib.html#importlib.import_module).
> 
> It's not quite clear to me what makes this "advanced", but it would probably be better to switch to `importlib.import_module`.

ok~